scripts/build.sh

```diff
@@ -42,8 +42,15 @@ else
     echo "Plugins Disabled"
 fi
 
+if [ "$FASTBUILD" == "yes" ];then
+  # Enable the `go build -i` flag to install dependencies during build and
```

Comment:
Is it allowed during build? My understanding is no network calls during build

Reply:
This flag installs the intermediate dependencies that `go build` builds into the `$GOPATH/pkg`. This doesn't actually download any packages.

Without this flag, on each build `go build` builds all intermediate packages, and discards them once the main package is built. This slows down subsequent builds which makes developement slower.

But when building RPMs, you build only once in a clean environment. So there is nothing to gain by trying to install the intermediate dependencies. Also, when building using system packaged dependencies, this flag will cause `go build` to try to install the built packages into system paths (`/usr/`), which fails when building as an unprivileged user.

Reply:
got it.